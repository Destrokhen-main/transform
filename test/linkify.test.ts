import transform from '../src/transform';

describe('Linkify', () => {
    it('should not linkify .cloud tld without linkifyTlds option', () => {
        const {
            result: {html},
        } = transform('yandex.cloud');

        expect(html).toMatchSnapshot();
    });
    it('should linkify .cloud tld with linkifyTlds option', () => {
        const {
            result: {html},
        } = transform('yandex.cloud', {linkifyTlds: 'cloud', linkify: true});

        expect(html).toMatchSnapshot();
    });
});
