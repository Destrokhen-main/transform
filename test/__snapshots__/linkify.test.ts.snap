// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Linkify should linkify .cloud tld with linkifyTlds option 1`] = `
<p>
  <a href="http://yandex.cloud">
    yandex.cloud
  </a>
</p>
`;

exports[`Linkify should not linkify .cloud tld without linkifyTlds option 1`] = `
<p>
  yandex.cloud
</p>
`;
